README.md

```diff
@@ -1,37 +1,106 @@
 # HStream Operator
 
-The purpose of the project is to simplify the deployment of the HStream cluster.
+## Introduction
 
-## Description
+HStream Operator is a Kubernetes operator designed to manage and maintain the HStreamDB cluster within a Kubernetes environment. The HStreamDB cluster comprises of several components including:
 
-A HStream cluster consists of HServer, HStore, HMeta and AdminServer.
+- [HServer](https://docs.hstream.io/reference/architecture/hserver.html)
+- [HStore](https://docs.hstream.io/reference/architecture/hstore.html)
+- HMeta
+- AdminServer
 
-## Getting Started
+It simplifies the deployment, scaling, and operations of HStreamDB clusters on Kubernetes, making it easier for users to manage their HStream components effectively. We use and get benefits from [kubebuilder](https://book.kubebuilder.io/) to simplify the development of the operator.
 
-You’ll need a Kubernetes cluster to run against. You can use [KIND](https://sigs.k8s.io/kind) to get a local cluster for testing, or run against a remote cluster.
+## Installation
 
-**Note:** Your controller will automatically use the current context in your kubeconfig file (i.e. whatever cluster `kubectl cluster-info` shows).
+We recommend using the [Helm](https://helm.sh/) package manager to install the HStreamDB operator on your Kubernetes cluster.
 
-### Running on the cluster
+> Currently, we haven't released the chart because of this operator is still at an early stage. So you
+> need to clone this repo and install the chart from the local directory.
+
+```sh
+git clone https://github.com/hstreamdb/hstream-operator.git && cd hstream-operator
+helm install hstream-operator deploy/charts/hstream-operator -n hstream-operator-system --create-namespace
+```
 
-1. Install Instances of Custom Resources:
+Every releases will be published to [GitHub Releases](https://github.com/hstreamdb/hstream-operator/releases), you
+can also install the operator with the following command:
 
 ```sh
-kubectl apply -f config/samples/
+kubectl apply -f https://github.com/hstreamdb/hstream-operator/releases/download/0.0.8/hstream-operator.yaml
```

Comment:
please check here again. In my test, need to add `--server-side` option, otherwise the operator will fail

Reply:
The server-side apply can benefit somewhere like getting the fields accurately that other users have overridden. It's not necessary to be used. In my opinion, it's more of a choice to use depending on the situation.

Reply:
at least in my local test
![image](https://github.com/hstreamdb/hstream-operator/assets/45479280/c2c235f7-00db-48a3-b68c-1c14eceb5dd1)